certbot/tests/renewal_test.py

```diff
@@ -0,0 +1,30 @@
+"""Tests for certbot.renewal"""
+import os
+import mock
+import unittest
+import tempfile
+
+from certbot import configuration
+from certbot import renewal
```

Comment:
Can you move this into the method, just before calling `_restore_webroot_config`?